react/index.tsx

```diff
@@ -73,7 +73,8 @@ const OrderPlaced: FC = () => {
 
           <ExtensionPoint id="op-header" />
 
-          <main
+          <div
```

Comment:
Do you think of this as major or minor? I see some custom css made with html selectors being lost here possibly.

Reply:
Could be, yeah, BUT we don't allow to target elements (i.e `main`) in our css overrides, the only way is with our documented css handles!